scripts/build-homepage

```diff
@@ -20,8 +20,9 @@ kdb --version
 # prevent jenkins from killing daemons
 BUILD_ID=dontKillMe
 
-# then restart the backend
+# then restart the backend (sleep prevents 'address already in use')
 kdb stop-rest-backend
+sleep 3
```

Comment:
`service.backlog` is for clients connecting to the server, not for the server itself. I think you shouldn't worry about 3 seconds downtime...